scipy/sparse/linalg/eigen/lobpcg/lobpcg.py

```diff
@@ -127,7 +127,7 @@ def _get_indx(_lambda, num, largest):
 
 def lobpcg(A, X,
            B=None, M=None, Y=None,
-           tol=None, maxiter=20,
+           tol=None, maxiter=None,
```

Comment:
In your `svds` PR, was `maxiter=None` being passed in? That's why this was failing?

Reply:
yes